### DISCUSSION TITLE
Fontconfig Error: Issue with CascadiaCode.ttf Handling in osgearth Installation

### DISCUSSION THREAD

**Discussion by zhgzs666** [pelicanmapping/osgearth#2444] [Support Q&A]
I am a novice with osgearth. When I followed the Installation on the official website and built myApp.exe using the Sample CMakeLists.txt and Sample main.cpp, Fontconfig generated the following error: 'Error reading file C:/Windows/fonts\CascadiaCode.ttf: file not handled.' I searched for related solutions and modified the FONTCONFIG_FILE environment variable, but the problem still persists. How can I solve this issue?

**Comment by gwaldron** [comment]
This warning comes from OpenSceneGraph and I do not know what the problem is.
Unless you're having trouble with text objects, I would ignore it.
